modules/layers/src/geojson-layer/geojson-layer.ts

```diff
@@ -197,6 +197,13 @@ type _GeoJsonLayer3DProps<DataT> = {
    */
   wireframe?: boolean;
 
+  /**
+   * (Experimental) This prop is only effective with `XYZ` data.
+   * When true, polygon tesselation will be performed on the plane with the largest area, instead of the xy plane.
+   * @default false
+   */
+  _full3d?: boolean;
```

Comment:
Add to the documentation?

Reply:
Have taken a stab at it in the latest commit, take a look and say if you think it needs more fleshing out